guides/rhai_expressions_cheat_sheet.cheatmd

```diff
@@ -0,0 +1,54 @@
+# Rhai expressions cheatsheet
+
+We are grouping into this cheatsheet the most frequent manipulations that can be done through Rhai, for convenience.
+
+## Cheatsheet
+{: .col-2}
```

Comment:
I think this is not rendering properly in my markdown renderer, is this expected?

Reply:
Yes :smile: